[2023-05-29T09:35:55.797+0000] {taskinstance.py:1093} INFO - Dependencies not met for <TaskInstance: stock_market_dag.process_raw_data_etfs scheduled__2023-05-28T00:00:00+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2023-05-29T09:35:55.798+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
